src/app/articles/articles.component.html

```diff
@@ -1,8 +1,9 @@
 <mat-list>
```

Comment:
Maybe turn the list into another `mat-card` with an action button (CREATE) at the bottom to that goes to the news creator?

Reply:
Are you referring to the news page? This component is put in a card on the dashboard page and just displayed on the news page. I could make it into a card on the news page.

Reply:
Yep to the News page here:
![screen shot 2018-10-18 at 9 52 35 pm](https://user-images.githubusercontent.com/437977/47198276-2e0fc600-d320-11e8-8884-033ddda1b7f3.png)

I'm suggesting putting the list of news items into a `mat-card`....maybe something like this:
![screen shot 2018-10-18 at 9 54 27 pm](https://user-images.githubusercontent.com/437977/47198335-79c26f80-d320-11e8-8697-c57c9ccab419.png)
with a `CREATE` action at the bottom like:
![screen shot 2018-10-18 at 9 55 28 pm](https://user-images.githubusercontent.com/437977/47198375-aaa2a480-d320-11e8-9e08-39510ff85f95.png)
Does that make sense?